R/save.r

```diff
@@ -13,6 +13,16 @@
 #' containing the `%` sign, use `%%`. For example, `filename = "figure-100%%.png"`
 #' will produce the filename `figure-100%.png`.
 #'
+#' @section Writing to a graphics device:
+#'
+#' In most cases `ggsave()` is the simplest way to save your plot, but
+#' sometimes it is necessary to save the plot by writing directly to a
+#' graphics device. For example, to create a PNG file, you could first
+#' open the device with a command like `png("mtcars.png")`. Next, you
+#' would create the ggplot with something like
+#' `ggplot(mtcars, aes(mpg, wt)) + geom_point()` and then use `dev.off()`
```

Comment:
Should this state explicitly that the plot needs to be printed? `p <- ggplot(...); print(p)`

Reply:
Oops, yes. The revised version does this explicitly in the examples section